x/tokenomics/keeper/token_logic_modules.go

```diff
@@ -194,6 +194,21 @@ func (k Keeper) ProcessTokenLogicModules(
 	}
 
 	// Execute all the token logic modules processors
+	// TODO_CRITICAL: Per-claim validator reward distribution causes significant precision loss.
```

Comment:
Can you please:

1. Move this into a GitHub issue
2. Link to the GitHub issue here (and keep this shorter)
3. Provide an example with some numbers in the issue

Reply:
- https://github.com/pokt-network/poktroll/issues/1758